vxyowsup/whatsapp.py

```diff
@@ -124,8 +132,11 @@ def __init__(self, transport):
         super(WhatsAppInterface, self).__init__()
         self.transport = transport
 
-    def send_to_human(self, text, to_address):
+    def send_to_human(self, text, to_address, message_id):
+        # message_id is vumi id
         message = TextMessageProtocolEntity(text, to=to_address)
+        self.transport.redis.setex(message.getId(), self.transport.config.ack_timeout, message_id)
```

Comment:
We could create the whatsapp message in the `handle_outbound` method of the transport